content/pages/faq.md

```diff
@@ -19,13 +19,36 @@ Holding events which are only open to women helps some women deal with
 [stereotype threat](http://geekfeminism.wikia.com/wiki/Stereotype_threat
 "Geek Feminism Wiki: Stereotype Threat").
 
-An example of an event that's women-only is our mentoring program, where we
-match up junior female students with more senior female students to talk about
-their experiences and support them throughout their academic career.  Women can
-obviously seek out male mentors as well, but they don't quite share the same
-experiences so the impact of the mentoring is not the same. There are also
-events that are traditionally very competitive (such as hackathons) that women
-feel more comfortable coming out to if they're women-only.
+There are a few categories of events that the Undergraduate Committee
+generally chooses to keep women only. These include: workshops,
+networking, and menotring or discussion circle-style events.
+
+Women are often excluded from these kinds of events when held by other
+organizations, sometimes because of unconcious biases playing a part 
+in the admissions process, or because some women do not feel comfortable
+at this kind of event when they know the majority of attendants will
```

Comment:
this kind of event -> these kinds of events

(minor nit: you use "these kind of events" twice in the same sentence, so you could reword it but it's not necessary)